examples/customresources/demos/complexmodules/.gitignore

```diff
@@ -0,0 +1,2 @@
+gizmomodule/gizmomodule
```

Comment:
This new `complexmodules` directory is certainly a bit confusing with the existing `complexmodule` one. At the least, I think a README in `complexmodules` could be helpful if we're really offering this as an example of Golang modules.

Reply:
Agree... name it "multiple-modules" or something instead?

Reply:
renamed to `multiplemodules`, added another section to the README to explain what it does